tests/sentry/api/endpoints/test_organization_details.py

```diff
@@ -353,14 +354,17 @@ def test_various_options(self, mock_get_repositories):
             "defaultRole": "owner",
             "require2FA": True,
             "allowJoinRequests": False,
+            "providerKey": "google",
+            "providerConfig": {"domain": "foo.com"},
         }
 
         # needed to set require2FA
         interface = TotpInterface()
         interface.enroll(self.user)
         assert self.user.has_2fa()
 
-        self.get_success_response(self.organization.slug, **data)
+        with self.feature("organizations:api-auth-provider"):
```

Comment:
You should use the decorator to set the feature flag, so it is more obvious that this test case is affected by the feature flag.

Reply:
You can remove this after you've set the decorator.